### PR TITLE
Implemented Area Fill saving

### PR DESCRIPTION
**Pull request by Jossos**
- Implemented Area Fill saving
- Fixes [AB#1716](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/1716)

### CODE REVIEW

File: src/scene/vcPOI.cpp [hunk 1]
```diff
@@ -497,9 +558,10 @@ void vcPOI::OnNodeUpdate(vcState *pProgramState)
   vdkProjectNode_GetMetadataUint(m_pNode, "backColour", &m_backColour, vcIGSW_ImGuiToBGRA(pProgramState->settings.tools.label.backgroundColour));
   vdkProjectNode_GetMetadataUint(m_pNode, "lineColourPrimary", &m_line.colourPrimary, vcIGSW_ImGuiToBGRA(pProgramState->settings.tools.line.colour));
   vdkProjectNode_GetMetadataUint(m_pNode, "lineColourSecondary", &m_line.colourSecondary, 0xFFFFFFFF);
+  vdkProjectNode_GetMetadataUint(m_pNode, "measurementAreaFillColour", &m_measurementAreaFillColour, vcIGSW_ImGuiToBGRA(pProgramState->settings.tools.line.colour));
   vdkProjectNode_GetMetadataString(m_pNode, "hyperlink", &pTemp, "");
   udStrcpy(m_hyperlink, pTemp);
-
+
```
Comment (samsinsane) [inline src/scene/vcPOI.cpp:564]: This should be reverted too.

File: src/scene/vcPOI.cpp [hunk 2]
```diff
@@ -43,9 +43,7 @@ class vcPOIState_General
 public:
   vcPOIState_General(vcPOI *pParent)
     : m_pParent(pParent)
-  {
-
-  }
+  {}
```
Comment (samsinsane) [inline src/scene/vcPOI.cpp:46]: This seems like an unnecessary change? Is this consistent with other constructors now?

File: src/scene/vcPOI.cpp [hunk 3]
```diff
@@ -497,9 +558,10 @@ void vcPOI::OnNodeUpdate(vcState *pProgramState)
   vdkProjectNode_GetMetadataUint(m_pNode, "backColour", &m_backColour, vcIGSW_ImGuiToBGRA(pProgramState->settings.tools.label.backgroundColour));
   vdkProjectNode_GetMetadataUint(m_pNode, "lineColourPrimary", &m_line.colourPrimary, vcIGSW_ImGuiToBGRA(pProgramState->settings.tools.line.colour));
   vdkProjectNode_GetMetadataUint(m_pNode, "lineColourSecondary", &m_line.colourSecondary, 0xFFFFFFFF);
+  vdkProjectNode_GetMetadataUint(m_pNode, "measurementAreaFillColour", &m_measurementAreaFillColour, vcIGSW_ImGuiToBGRA(pProgramState->settings.tools.line.colour));
```
[the remark this answers is not in the document]
Comment (Jossos) [reply src/scene/vcPOI.cpp:561]: it was in the wrong spot i forgot to remove the comment
